src/style.css

```
@@ -0,0 +1,131 @@
+/* #region reset */
+
+* {
+  box-sizing: border-box;
+}
+
+body, h1, h2, h3, ul, p {
+  margin: 0;
+  padding: 0;
+}
+
+/* #endregion */
+
+/* #region overall */
+
+html {
+  font-size: 12px;
+  font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+
+/* #endregion */
+
+/* #region product-card */
+
+.product-card {
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  padding: 32px 16px 0px;
+}
+
+.product-card__img {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.product-card__img img {
```

```suggestion
.product-card__img {
```
Думаю так тоже можно написать